BurnOutSharp/PackerType/PECompact.cs

```diff
@@ -30,9 +30,13 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             return MatchUtil.GetFirstMatch(file, fileContent, matchers, includePosition);
         }
 
+        // TODO: Improve version detection, Protection ID is able to detect ranges of versions. For example, 1.66-1.84 or 2.20-3.02.
         public static string GetVersion(string file, byte[] fileContent, List<int> positions)
         {
-            return $"v{BitConverter.ToInt16(fileContent, positions[0] + 4)}";
+            short version = BitConverter.ToInt16(fileContent, positions[0] + 4);
+            if (version == 0)
+                return "PE Compact 2";
```

Comment:
If I remember my own code well enough, what you have here would end up with `PE Compact 2 PE Compact 2` if you had v0. Did you mean to have it return a blank string instead (so that it just displays `PE Compact 2` on output.